docs/USING_ADVANCED.md

```diff
@@ -1,3 +1,23 @@
+## Marked instance
+
+By default marked stores options and extensions in the global scope. In node js when you import marked into two scripts and change the options in one script the options will be changed in the other script.
+
+You can create an instance of marked to keep options and extensions locally scoped.
+
+```js
+import { Marked } from 'marked';
+const marked = new Marked([...useArgs]);
+```
+
+```js
+import { marked } from 'marked';
+const myMarked = new marked.Instance([...useArgs]);
```

Comment:
I'm not sure this usage is necessary. Is there a reason to use this instead of the one above?

Reply:
I agree. If we provide multiple ways to do the same thing, that's just an additional point of confusion for users asking what the difference is.

Reply:
I changed it to show a script tag so people know this is for client side javascript

Reply:
Ya I agree we can remove this for now and add it later if someone has a valid use case.